src/amber/websockets/client_sockets.cr

```diff
@@ -36,8 +36,8 @@ module Amber
       private def heartbeat
         spawn do
           @@heartbeat_started = true
-          @@client_sockets.each_value(&.beat)
           sleep BEAT_INTERVAL
+          @@client_sockets.each_value(&.beat)
           heartbeat
```

Comment:
I'm wondering if this will exit the previous `spawn`, or if this is going to get stacked up